src/output.jl

```diff
@@ -210,7 +211,7 @@ end
 # IJulia
 # ---------------------------------------------------------
 
-const _ijulia_output = String["text/html"]
```

Comment:
yeah of course, that's why it's a "do not merge" ;) These edits are just there to make my life a bit easier temporarily!